apps/library/lib/library/dictionary.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Library.Dictionary do
+  use GenServer
+
+  def start_link(state, opts \\ []) do
+    {:ok, dictionary} = GenServer.start(__MODULE__, state, opts)
+  end
+
+  def check_word(word) do
+    GenServer.call(:dictionary, {:check_word, String.downcase(word)})
+  end
+
+  ### Server callbacks
+
+  def init([]) do
+    {:ok, words} = File.read("/usr/share/dict/words")
```

Comment:
More as a comment thing (and fancy new syntax that I try to find places to use) this could be written with the new [with](http://elixir-lang.org/docs/stable/elixir/Kernel.SpecialForms.html#with/1) special form syntax.

```elixir
with {:ok, words} <- File.read("/usr/share/dict/words"),
     do: {:ok, format_words(words)}
```

You don't have to change this, I just really like with `with` haha.